Cynosura.Studio.Web/Program.cs

```diff
@@ -43,6 +43,19 @@ public static void Main(string[] args)
         public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
             WebHost.CreateDefaultBuilder(args)
                 .UseStartup<Startup>()
+                .ConfigureAppConfiguration((hostingContext, config) =>
```

Comment:
Why need to move this to Program.cs?

Reply:
This is from the best practices. WebHostingBuilder defining configuration by itself and then integrates it into the Startup class. When we decide to add integration tests to the project, this will be required.